portal/footer.ms

```diff
@@ -11,5 +11,8 @@
 
   <!-- Scripts -->
   <script src="assets/js/global.js"></script>
+  {{#theme}}
+  <script src="assets/themes/{{theme}}/global.js"></script>
```

Comment:
So every theme has to implement Javascript or the request for this element will fail.